src/Draco.Compiler/Internal/Symbols/IOverridableSymbol.cs

```diff
@@ -0,0 +1,13 @@
+namespace Draco.Compiler.Internal.Symbols;
+
+/// <summary>
+/// Represents any symbol that can be overridden.
+/// </summary>
+internal interface IOverridableSymbol
+{
+    /// <summary>
+    /// Represents the explicitly overridden symbol.
+    /// Not null, if this symbol has a different signature than the symbol it overrode.
+    /// </summary>
+    public Symbol? ExplicitOverride { get; }
```

Comment:
Why not have this always contain the overridden symbol? It could be quite useful to have an `Overrides` property in general, and optionally have a boolean denoting, if the override is explicit or something.